src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -174,15 +200,12 @@ private async Task<int> StartMonitor()
             Task monitorTask = new Task(() => {
                 try
                 {
-                    var configuration = new SessionConfiguration(
-                        circularBufferSizeMB: 1000,
-                        format: EventPipeSerializationFormat.NetTrace,
-                        providers: Trace.Extensions.ToProviders(providerString));
-
-                    var binaryReader = EventPipeClient.CollectTracing(_processId, configuration, out _sessionId);
-                    EventPipeEventSource source = new EventPipeEventSource(binaryReader);
-                    source.Dynamic.All += Dynamic_All;
-                    source.Process();
+                    RequestTracingV2(providerString);
+                }
+                catch (EventPipeUnknownCommandException)
+                {
+                    // If unknown command exception is thrown, it's likely the app being monitored is running an older version of runtime that doesn't support CollectTracingV2. Try again with V1.
+                    RequestTracingV1(providerString);
```

Comment:
I'm ok to submit this as-is to make the preview7 cut-off without retesting, but I would open a quick issue to revisit the error handling here. If RequestTracingV1 had an error it is not covered by the try region that RequestTracingV2 has. In particular, it would not call Debug.WriteLine below.